yeonjae/programmers-2/스킬트리.js

```diff
@@ -0,0 +1,40 @@
+function solution(skill, skill_trees) {
+  const filteredSkillTrees = [];
+  let answer = 0;
+
+  skill_trees.forEach((el) => {
+    let str = "";
+
+    for (let i = 0; i < el.length; i++) {
+      const idx = skill.indexOf(el[i]);
+
+      if (idx !== -1) {
+        str += el[i];
+      }
+    }
+    filteredSkillTrees.push(str);
+  });
+
+  filteredSkillTrees.forEach((el) => {
+    let checkValid = true;
```

Comment:
```suggestion
    let isValid = true;
```
- isValid가 나은 것 같습니다!

Reply:
넵 확인했습니다!